backend/copr_backend/storage.py

```diff
@@ -206,6 +267,60 @@ def delete_repository(self, chroot):
         self.client.delete_repository(repository)
         self.client.delete_distribution(distribution)
 
+    def delete_project(self, dirname):
+        prefix = "{0}/{1}".format(self.owner, dirname)
+        response = self.client.list_repositories(prefix)
+        repositories = response.json()["results"]
+        for repository in repositories:
+            self.client.delete_repository(repository["pulp_href"])
+
+    def delete_builds(self, dirname, chroot_builddirs, build_ids):
+        # pylint: disable=too-many-locals
+        result = True
+        for chroot, subdirs in chroot_builddirs.items():
+            if chroot == "srpm-builds":
```

Comment:
this is because repodata aren't created for srpm build thus we don't need to delete anything for srpm builds?

Reply:
I am wondering what will happen with logs? We delete them in the backend storage but not in pulp - is that part of `delete artifacts`? Then the logs will still remain

Reply:
> this is because repodata aren't created for srpm build thus we don't need to delete anything for srpm builds?

(So far) we don't upload results of source builds to Pulp, so there is nothing to remove. I added a code comment so it's clear.

However, in that case, we probably need to remove them from the backend storage, right?

Reply:
I've got srpm stored inside the pulp instance:
```
lftp localhost:/pulp/content/nikromen/test-pulp-3/fedora-39-x86_64/Packages/p> ls
drwxr-xr-x  --  ..
-rw-r--r--  --  python-dside-2.3.3-1.fc39.src.rpm
-rw-r--r--  --  python3-dside-2.3.3-1.fc39.noarch.rpm
lftp localhost:/pulp/content/nikromen/test-pulp-3/fedora-39-x86_64/Packages/p>
```
but it is somehow deleted with the build deletion - probably the resources maps also the srpms.

> However, in that case, we probably need to remove them from the backend storage, right?

I only checked the existence of the project inside pulp - after meeting I'll review backend storage if there are some remnants

Reply:
> after meeting I'll review backend storage if there are some remnants

yes, there are remnants from the deleted builds and deleted projects - so we need to temporarily call the delete methods from backend storage inside pulp storage to remove what's on backend.

Reply:
Done